.travis.pom.xml

```diff
@@ -0,0 +1,608 @@
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <excludeFilterFile>${basedir}/../findbugs-exclude.xml</excludeFilterFile>
+          <plugins>
+            <plugin>
+              <groupId>com.google.http-client</groupId>
+              <artifactId>google-http-client-findbugs</artifactId>
+              <version>${project.http.version}</version>
+            </plugin>
+          </plugins>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>clirr-maven-plugin</artifactId>
+        <configuration>
+          <comparisonVersion>1.19.0</comparisonVersion>
+          <ignoredDifferencesFile>${basedir}/../clirr-ignored-differences.xml</ignoredDifferencesFile>
+          <logResults>true</logResults>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>animal-sniffer-maven-plugin</artifactId>
+        <configuration>
+          <signature>
+            <groupId>org.codehaus.mojo.signature</groupId>
+            <artifactId>java15</artifactId>
+            <version>1.0</version>
+          </signature>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.http.version>1.24.0-SNAPSHOT</project.http.version>
+    <project.oauth.version>1.24.0-SNAPSHOT</project.oauth.version>
+    <project.jsr305.version>3.0.2</project.jsr305.version>
+    <project.gson.version>2.1</project.gson.version>
+    <project.jackson-core-asl.version>1.9.11</project.jackson-core-asl.version>
+    <project.jackson-core2.version>2.9.2</project.jackson-core2.version>
+    <project.protobuf-java.version>2.6.1</project.protobuf-java.version>
+    <project.guava.version>19.0</project.guava.version>
+    <project.appengine.version>1.7.7</project.appengine.version>
+    <project.xpp3.version>1.1.4c</project.xpp3.version>
+    <project.commons-logging.version>1.1.1</project.commons-logging.version>
+    <project.httpclient.version>4.5.3</project.httpclient.version>
+    <project.httpcore.version>4.0.1</project.httpcore.version>
+    <project.jdo2-api.version>2.3-eb</project.jdo2-api.version>
+    <project.transaction-api.version>1.1</project.transaction-api.version>
```

Comment:
This looks like a maintenance nightmare - we have to make sure versions match in both the main pom and this travis pom. What was the reason for a separate pom?

Also, have you considered trying CircleCI instead?

Reply:
Good point. 

CircleCI seems awesome. I wonder if there wouldn't be the same problem. Newer versions of the Android SDK hide that Play Services jar inside an .aar. The best approach would probably be to make testing google-api-java-client-samples part of the integration and build with Gradle.